### PR TITLE
Test build with G++ 11

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

**Comment by geoffthemedio** [comment]
I don't have any relevant comment...

File: .github/workflows/build-windows.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v1.0.2
         with:
-          vs-version: '[16.0,16.10)'
+          vs-version: '[16.0,16.30)'
```
Comment (geoffthemedio) [inline .github/workflows/build-windows.yml:37]: > I suppose only the newest MSVS installed on CI so I couldn't choose older one.

What does this line do then?

Comment (o01eg) [reply .github/workflows/build-windows.yml:37]: Searches any msbuild with that version and add it to `PATH`.

Comment (geoffthemedio) [reply .github/workflows/build-windows.yml:37]: I meant what does changing from `.10` to `.30` do?

Comment (o01eg) [reply .github/workflows/build-windows.yml:37]: To allow `MSVC 19.29.30037.0` be included.

Comment (geoffthemedio) [reply .github/workflows/build-windows.yml:37]: If "only the newest" version is installed, why do you need to specify a version at all?

Or is it the newest version of MSVC2019 and you want to specify not MSVC2017?

Comment (o01eg) [reply .github/workflows/build-windows.yml:37]: > Or is it the newest version of MSVC2019 and you want to specify not MSVC2017?

Yes.

Comment (geoffthemedio) [reply .github/workflows/build-windows.yml:37]: Makes sense then.

Otherwise, as above,
> I don't have any relevant comment...

**Pull request merged**